### ISSUE TITLE
Tags are not stripped from og:description

### ISSUE THREAD

**Issue by peterjaap** [Yoast/Yoast-SEO-for-Magento2#62]
So we get stuff like this;

![image](https://user-images.githubusercontent.com/431360/35390659-7cf859fa-01dc-11e8-9f2d-041278f7c64d.png)

... etc

Because the HTML is closed, the actual text shows on top of the page.

We now have overriden `vendor/maxserv/magento-module-yoastseo/view/frontend/templates/yoastseo.phtml` in our theme and changed;

`<meta property="og:description" content="<?php echo $meta->getOpenGraphDescription(); ?>"/>`

to;

`<meta property="og:description" content="<?= $block->escapeHtmlAttr($block->stripTags($meta->getOpenGraphDescription())) ?>"/>`

**Comment by peterjaap** [comment]
That produced ugly output. Better yet, we created plugins for the three MetaProviders that return nice output. One example;

```
<?php

namespace Elgentos\FixYoastSeo\Plugin\Product;

class MetaProviderPlugin
{
    public function afterGetOpenGraphDescription(\MaxServ\YoastSeo\Model\EntityConfiguration\Catalog\Product\MetaProvider $subject, $result)
    {
        return trim(preg_replace('#<[^>]+>#', ' ', $result));
    }
}
```

**Comment by vincenthornikx** [comment]
issue has been fixed in 2.0.0
